rskj-core/src/main/java/co/rsk/core/bc/BlockChainImpl.java

```diff
@@ -144,6 +144,9 @@ public ImportResult tryToConnect(Block block) {
             }
 
             try {
+                org.slf4j.MDC.put("blockHash", block.getHash().toHexString());
+                org.slf4j.MDC.put("blockHeight", Long.toString(block.getNumber()));
```

Comment:
Wouldn't be better if we wrap-up org.slf4j.MDC with an own implementation? Kind of helper or sthg?